index-test.html

```diff
@@ -115,6 +123,21 @@ <h3>Star Rating</h3>
           on-choose-rating="onChooseRating({rating})"
         ></app-star-rating>
       </div>
+
+      <div class="section">
+        <h3>Stat Card</h3>
+        <div class="section__stat-cards">
+          <app-stat-card
+            border-color="border-fusion-yellow"
+            border-color-selected="border-fusion-yellow--selected"
+            title-color="text-fusion-yellow"
```

Comment:
There's a bit of finesse with components and bindings and how customizable can they be vs. clarity. Too many bindings can be confusing + having three separate CSS class bindings means I have to store these on the consumer. What @JStein92 suggested could be an option where you just inject the color but for these stat cards, I rather have the component library store the colors when it makes sense.

Checkout some best practices online and if there's nothing better -- change it to the following (I'm open to push back if you feel something is better). 

Make multiple components. E.g. app-stat-card-warn, app-stat-card-danger, ...

*-warn stores the yellow colors using it's own CSS and gives us consistency so consumers don't have to worry about coloring.
 
E.g.
<app-start-card-warn 
            icon-class="fa fa-users"
            card-title="Pending Cases"
            on-select-show-me="updateDashboardCases('cases_pending')"
            stat-number-to-display="{{userNum}}"
            is-selected="statusToFilterOn === 'cases_pending'">
</app-stat-card>

<app-stat-card-info
 ....
 >
 
 I'm borrowing the naming convention from bootstrap 
 -warn => generally yellow 
 -danger => generally red 
 -success => generally green 
 -info => generally blue.
 -default => generally neutral colors?